gnovm/pkg/gnolang/store_test.go

```diff
@@ -0,0 +1,472 @@
+package gnolang
```

Comment:
Nitpick: why not run these tests in parallel?

Reply:
because I don't sign up to the "all tests should be parallel" church, but that's a discussion for another time :)

parallelized